### DISCUSSION TITLE
4.10.7 has just been released, let's move on

### DISCUSSION THREAD

**Discussion by cprudhom** [chocoteam/choco-solver#830] [Announcements]
Help us build the roadmap for the next version of choco-solver and be part of this great adventure !
The library may seem complex in some ways (and it is) but there are many ways to improve it.
We will be happy to accompany you !

**Comment by dimitri-justeau** [comment]
Thank you to involve us in this process, I will try to share my ideas for the next versions of choco-solver :-)
